Linear/main_LinEl.C

```diff
@@ -160,9 +162,20 @@ int main (int argc, char** argv)
         adaptor = atoi(argv[i]+5);
     }
     else if (!infile)
+    {
       infile = argv[i];
+      if (strcasestr(infile,".xinp"))
+        if (!args.readXML(infile,false))
+          return 1;
+      if(args.dim == 1)
+        oneD = true;
+      else if(args.dim == 1)
+        twoD = true;
+    }
     else
+    {
```

Comment:
heh. nope. if i cannot brace my multi-line if's you cannot brace your single-line else ;P